pkgs/development/python-modules/pyrdfa3/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchpatch
+, isPy27
+, rdflib
+, html5lib
+}:
+
+buildPythonPackage rec {
+  pname = "pyRdfa3";
+  version = "3.5.3";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
```

Which in turn requires a fixup here.

```suggestion
    pname = "pyRdfa3";
    inherit version;
```